src/common/SPELLS_PALADIN.js

```diff
@@ -94,6 +94,11 @@ export default {
     icon: 'ability_paladin_lightofthemartyr',
     manaCost: 16500,
   },
+  MARAADS_DYING_BREATH_BUFF: {
```

Comment:
As far as I can see, no other spell with this name, so just call it `MARAADS_DYING_BREATH` - it's distinguished from the item by being `SPELLS.` rather than `ITEMS.`

Reply:
While true, suffixing it with `_BUFF` leaves no doubt about this and thus adds value imo.

Reply:
While I've seen different ways of doing this, I thought the style thus far has been "always use the spell's exact name unless forced not too" e.g. when two spells have the same name.

Reply:
or I should say "the style is _supposed to be_"

Reply:
None of the rules are 100% hard (so they're more *guidelines*), if deviating from it makes things clearer then doing so is encouraged.

Reply:
ANARCHY CONFIRMED

Reply:
*\*sirens in the background**